cmds/report.c

```diff
@@ -221,6 +254,7 @@ static void report_functions(struct uftrace_data *handle, struct opts *opts)
 	struct rb_root name_root = RB_ROOT;
 	struct rb_root sort_root = RB_ROOT;
 	const char f_format[] = "  %10.10s  %10.10s  %10.10s  %-.*s\n";
+	const char csv_format[] = "  %10.10s,  %10.10s,  %10.10s,  %-.*s\n";
```

Comment:
I prefer the following output rather than having extra formating.
```
$ uftrace report --format=csv
Total time, Self time, Calls, Function
3.240 us, 0.486 us, 1, main
2.754 us, 0.427 us, 1, a
2.327 us, 0.560 us, 1, b
1.767 us, 0.993 us, 1, c
1.507 us, 1.507 us, 1, __monstartup
0.943 us, 0.943 us, 1, __cxa_atexit
0.774 us, 0.774 us, 1, getpid
```

Reply:
I put a "," in the string. However, the broken string is output(`Total time` and `Function`).


```
  Total time  Self time,      Calls,  ^A
  3.518 us,  0.583 us,           1,  main
  2.935 us,  0.549 us,           1,  a
  2.386 us,  0.606 us,           1,  b
  1.780 us,  0.925 us,           1,  c
  1.172 us,  1.172 us,           1,  __monstartup
  0.977 us,  0.977 us,           1,  __cxa_atexit
  0.855 us,  0.855 us,           1,  getpid
```

```diff
diff --git a/cmds/report.c b/cmds/report.c
index b09d326..1c0cc9a 100644
--- a/cmds/report.c
+++ b/cmds/report.c
@@ -254,7 +254,6 @@ static void report_functions(struct uftrace_data *handle, struct opts *opts)
        struct rb_root name_root = RB_ROOT;
        struct rb_root sort_root = RB_ROOT;
        const char f_format[] = "  %10.10s  %10.10s  %10.10s  %-.*s\n";
-       const char csv_format[] = "  %10.10s,  %10.10s,  %10.10s,  %-.*s\n";
        const char line[] = "=================================================";

        build_function_tree(handle, &name_root, opts);
@@ -266,11 +265,11 @@ static void report_functions(struct uftrace_data *handle, struct opts *opts)

        if (opts->format) {
                if (avg_mode == AVG_NONE)
-                       pr_out(csv_format, "Total time", "Self time", "Calls", maxlen, "Function");
+                       pr_out(f_format, "Total time,", "Self time,", "Calls,",  "Function");
                else if (avg_mode == AVG_TOTAL)
-                       pr_out(csv_format, "Avg total", "Min total", "Max total", maxlen, "Function");
+                       pr_out(f_format, ",Avg total", ",Min total", ",Max total",  "Function");
                else if (avg_mode == AVG_SELF)
-                       pr_out(csv_format, "Avg self", "Min self", "Max self", maxlen, "Function");
+                       pr_out(f_format, ",Avg self", ",Min self", ",Max self",  "Function");

                print_and_delete(&sort_root, true, NULL, print_csv_function);
        }

```

Reply:
Fixed